addons/project/static/src/components/project_control_panel/project_control_panel.xml

```diff
@@ -38,8 +41,8 @@
     </t>
 
     <t t-name="project.ProjectControlPanel.Small" t-inherit="web.ControlPanel.Small" t-inherit-mode="primary" owl="1">
-        <xpath expr="//t[@t-call='web.Breadcrumbs.Small']" position="replace">
-            <t t-call="project.Breadcrumbs.Small"/>
+        <xpath expr="//div[hasclass('o_cp_bottom_left')]" position="inside">
+            <t t-call="project.ProjectControlPanelContent"/>
```

Comment:
> I understand you do this in order to move the badge button next to the "New" button. You have removed though the call to `project.Breadcrumbs.Small` and thus it can also be removed.

:+1: